README.md

```diff
@@ -69,9 +68,6 @@ To install the MapboxNavigation framework using [CocoaPods](https://cocoapods.or
 
 1. Create a [Podfile](https://guides.cocoapods.org/syntax/podfile.html) with the following specification:
    ```ruby
-   # Latest stable release
```

Comment:
Motivation: 6261ce794f802535dc56b5d9b5c071f291db43f9

Reply:
Yep, makes sense. I think we had removed it previously but it came back in during a merge from main.